cpu/nrf51822/periph/random.c

```diff
@@ -25,33 +26,60 @@
 /* guard file in case no random device was specified */
 #if RANDOM_NUMOF
 
+uint8_t _rng_int = 0;
```

Comment:
unused variable? Periph drivers should only allocate RAM if really really needed...